src/test/java/reposense/authorship/AnnotatorAnalyzerTest.java

```diff
@@ -10,13 +10,17 @@
     @Test
     public void noAnnotationTest() {
         config.setAnnotationOverwrite(false);
+        config.setSinceDate(BLAME_TEST_SINCE_DATE);
+        config.setUntilDate(BLAME_TEST_UNTIL_DATE);
```

Comment:
can re-evaluate if theres still any values in this 2 tests?

Reply:
The `blameTest.java` has no author tag `@@author`. Hence these two tests yields the same output and they are not directly testing on whether the `annotationOverwrite` functionality works correctly.

Reply:
If thats the case, can
* put up an issue
* remove this entire test file
* move `BLAME_TEST_SINCE_DATE` and `BLAME_TEST_UNTIL_DATE` to `FileAnalyzerTest` class

Reply:
I was wondering if the `annotationTest.java` file is initially meant to be used for the `AnnotatorAnalyzerTest` instead, as it contains the author tag and the file name matches up with the test name.

<img width="1006" alt="Screen Shot 2019-12-24 at 10 29 27 PM" src="https://user-images.githubusercontent.com/32864116/71418839-a05c0300-26a7-11ea-9168-344aa0b2d05c.png">